assets/src/edit-story/components/panels/textStyle/font.js

```diff
@@ -73,19 +73,14 @@ function FontControls({ selectedElements, pushUpdate }) {
                 ({ value: weight }) => weight
               );
 
-              // The default weight is 400 or empty if there are none available.
-              let defaultWeight = fontWeightsArr?.length ? 400 : '';
-              // If the font doesn't have 400 as an option, let's take the first available option.
-              if (
-                fontWeightsArr?.length &&
-                !fontWeightsArr.includes(defaultWeight.toString())
-              ) {
-                defaultWeight = fontWeightsArr[0];
-              }
               const newFontWeight =
-                fontWeightsArr && fontWeightsArr.includes(String(fontWeight))
-                  ? fontWeight
-                  : defaultWeight;
```

Comment:
This really needs a comment to explain what is going on here.

Reply:
Comments added. Basically it gets the nearest or same font-weights from font weights list